test/sm3_internal_test.c

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2021 The OpenSSL Project Authors. All Rights Reserved.
+ * Copyright 2021 UnionTech. All Rights Reserved.
+ *
+ * Licensed under the OpenSSL license (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+/*
+ * Internal tests for the SM3 module.
+ */
+
+#include <string.h>
+#include <openssl/opensslconf.h>
+#include "testutil.h"
+
+#ifndef OPENSSL_NO_SM3
+# include "internal/sm3.h"
+
+static int test_sm3(void)
+{
+    static const unsigned char input1 [] = {
```

Comment:
there should be no space between variable name and `[`.

Reply:
It has been modified!